src/app/features/location/address-edit/address-edit.component.ts

```diff
@@ -16,6 +18,7 @@ import { FaIconComponent } from "@fortawesome/angular-fontawesome";
   selector: "app-address-edit",
   standalone: true,
   imports: [
+    NgIf,
```

Comment:
I think ngIf is not used anymore now, as the gps component was extracted? (the new `@if` syntax doesn't require an import)

Reply:
Yes, you are right. Now, we don't need to import it. I will remove it from this component. Thanks!